packs/equipment/drake-rifle-acid.json

```
@@ -1,5 +1,5 @@
 {
-    "_id": "FU9tZ3neEPM8szay",
+    "_id": "vWopq4da4bLMQ2Y6",
```

```suggestion
    "_id": "FU9tZ3neEPM8szay",
```
Don't let IDs change, that will break any existing link to the item.